Collatz.java

```diff
@@ -1,6 +1,28 @@
 // Demonstrates the Collatz conjecture.
 public class Collatz {
 	public static void main(String args[]) {
-	    // Replace this comment with your code
+		int iterations = Integer.parseInt(args[0]);
+		boolean isVerbose = args[1].equals("v");
+
+		for (int i = 1; i <= iterations; i++) {
+			int seed = i;
+			int count = 1;
+
+			String verboseOutput = seed + " ";
+
+			do {
+				seed = seed % 2 == 0 ? seed / 2 : 3 * seed + 1;
```

Comment:
Small tip I would add () for both trueCase and false case 
to make it more clear